assets/components/LogContainer.vue

```diff
@@ -1,14 +1,26 @@
 <template>
   <scrollable-view :scrollable="scrollable" v-if="container">
     <template v-slot:header v-if="showTitle">
-      <div class="mr-0 columns is-vcentered is-hidden-mobile">
-        <div class="column is-clipped">
+      <div class="mr-0 columns is-vcentered is-marginless is-hidden-mobile">
+        <div class="column is-clipped is-paddingless">
           <container-title :container="container" @close="$emit('close')"></container-title>
         </div>
-        <div class="column is-clipped">
+        <div class="column is-clipped is-paddingless">
           <container-stat :stat="container.stat" :state="container.state"></container-stat>
         </div>
-        <div class="column is-narrow">
+        <div class="column is-clipped is-paddingless"></div>
+        <div class="column is-narrow is-paddingless" v-if="closable">
+          <button class="delete is-medium" @click="$emit('close')"></button>
+        </div>
+      </div>
+      <div class="mr-0 is-vcentered is-hidden-mobile">
+        <div class="is-flex">
+          <a @click="onClearClicked" class="button is-small is-outlined" id="clear">
+            <span class="icon">
+              <icon name="bin"></icon>
+            </span>
+          </a>
+          <div class="is-flex-grow-1"></div>
```

Comment:
So like I mentioned in my comment, having another toolbar makes the UI very clunky. I recommend looking in to centering a toolbar and using `position: fixed` to keep it floating.

Reply:
Used `position: absolute` to center it, which takes into account the side menu. Done.

![Screenshot 2021-11-04 at 20-42-35 si-db - Dozzle](https://user-images.githubusercontent.com/3276350/140439625-a462fbf3-bb35-454a-b0f7-ee1c10496b6d.png)